sklearn/tests/test_docstring_parameters.py

```diff
@@ -161,13 +167,82 @@ def test_tabs():
                                     % modname)
 
 
-@pytest.mark.parametrize('name, Classifier',
-                         all_estimators(type_filter='classifier'))
-def test_classifier_docstring_attributes(name, Classifier):
-    docscrape = pytest.importorskip('numpydoc.docscrape')
+@pytest.mark.parametrize('name, Estimator',
+                         all_estimators())
+def test_fit_docstring_attributes(name, Estimator):
+    pytest.importorskip('numpydoc')
     from numpydoc import docscrape
 
-    doc = docscrape.ClassDoc(Classifier)
+    doc = docscrape.ClassDoc(Estimator)
     attributes = doc['Attributes']
-    assert attributes
-    assert 'classes_' in [att.name for att in attributes]
+
+    IGNORED = {'ClassifierChain', 'ColumnTransformer', 'CountVectorizer',
+               'DictVectorizer', 'FeatureUnion', 'GaussianRandomProjection',
+               'GridSearchCV', 'MultiOutputClassifier', 'MultiOutputRegressor',
+               'NoSampleWeightWrapper', 'OneVsOneClassifier',
+               'OneVsRestClassifier', 'OutputCodeClassifier', 'Pipeline',
+               'RFE', 'RFECV', 'RandomizedSearchCV', 'RegressorChain',
+               'SelectFromModel', 'SparseCoder', 'SparseRandomProjection',
+               'SpectralBiclustering', 'StackingClassifier',
+               'StackingRegressor', 'TfidfVectorizer', 'VotingClassifier',
+               'VotingRegressor'}
+    if Estimator.__name__ in IGNORED or Estimator.__name__.startswith('_'):
+        pytest.skip("Estimator cannot be fit easily to test fit attributes")
+
+    est = Estimator()
+
+    if Estimator.__name__ == 'SelectKBest':
+        est.k = 2
+
+    if Estimator.__name__ == 'DummyClassifier':
+        est.strategy = "stratified"
```

Comment:
I think the following might work for a larger number of estimators,
```py
from sklearn.utils.estimator_checks import  _construct_instance, _set_checking_parameters

est = _construct_instance(Estimator)

_set_checking_parameters(est)
```
Not asking to do it now, I can change it in a follow up PR.